README.md

```diff
@@ -51,6 +51,18 @@ cmake ..
 make install
 ```
 
+Some special considerations for specific hosts:
+
+###### Archlinux
```

Comment:
Nit: This is how Arch Linux formats its own title.
```suggestion
###### Arch Linux
```

Reply:
Nice, thanks! I'll make sure to fix that in the wiki :-)